src/SyntaxHighlighter/include/SyntaxHighlighter/Cpp.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYNTAX_HIGHLIGHTER_CPP_H_
+#define SYNTAX_HIGHLIGHTER_CPP_H_
+
+#include <absl/container/flat_hash_map.h>
+#include <qregularexpression.h>
+
+#include <QSyntaxHighlighter>
+
+namespace orbit_syntax_highlighter {
+
+/*
+  This a syntax highlighter for cpp.
+
+  It derives from QSyntaxHighlighter, so check out QSyntaxHighlighter's
+  documentation on how to use it. There are no additional settings or
+  APIs.
+*/
```

Comment:
One more thing: Dimitry recently pointed out, that we usually use `//` also for block comments. I did this wrong in the assembly syntax highlighter. But we should probably follow that guidance.

Reply:
Do you want me to also change these things in X86Assembly?

Reply:
No. This was not my intention at all. Of course I won't stop you if you tackle this. But I already made a mental note to address this soonish.